src/chia_dialect.rs

```diff
@@ -1,30 +1,27 @@
-use std::collections::HashMap;
-
 use crate::allocator::{Allocator, NodePtr};
+use crate::core_ops::{op_cons, op_eq, op_first, op_if, op_listp, op_raise, op_rest};
 use crate::cost::Cost;
 use crate::dialect::Dialect;
 use crate::err_utils::err;
-use crate::f_table::{f_lookup_for_hashmap, FLookup};
-use crate::more_ops::op_unknown;
-use crate::operator_handler::OperatorHandler;
+use crate::more_ops::{
+    op_add, op_all, op_any, op_ash, op_concat, op_div, op_div_deprecated, op_divmod, op_gr,
+    op_gr_bytes, op_logand, op_logior, op_lognot, op_logxor, op_lsh, op_multiply, op_not,
+    op_point_add, op_pubkey_for_exp, op_sha256, op_softfork, op_strlen, op_substr, op_subtract,
+    op_unknown,
+};
 use crate::reduction::Response;
 
-const QUOTE_KW: [u8; 1] = [1];
-const APPLY_KW: [u8; 1] = [2];
-
-pub struct OperatorHandlerWithMode {
-    f_lookup: FLookup,
+pub struct ChiaDialect {
     strict: bool,
 }
 
-impl OperatorHandlerWithMode {
-    pub fn new_with_hashmap(hashmap: HashMap<String, Vec<u8>>, strict: bool) -> Self {
-        let f_lookup: FLookup = f_lookup_for_hashmap(hashmap);
-        OperatorHandlerWithMode { f_lookup, strict }
+impl ChiaDialect {
+    pub fn new(strict: bool) -> ChiaDialect {
```

Comment:
`mempool`?

Reply:
yeah, I'm planning a separate PR to rename strict-mode